recipes/snmf/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+snmf=$PREFIX/opt/$PKG_NAME-$PKG_VERSION
+mkdir -p $snmf
+mkdir -p $PREFIX/bin
+cp -r ./* $PREFIX/bin
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $PREFIX/bin/Snmf.pl
+rm -f $PREFIX/bin/Snmf.bak
+chmod +x $PREFIX/bin/snmf.sh
+chmod +x $PREFIX/bin/plink
+chmod +x $PREFIX/bin/install.command
+make $PREFIX/bin/install.command
```

Comment:
As you have it now, the installation process happens inside `$PREFIX/bin`. Instead, it should happen in the build dir (the cwd of this script) and then executables should be copied over from there.